src/RA_RichPresence.h

```diff
@@ -5,18 +5,22 @@
 #include "RA_Condition.h"
 #include "RA_MemValue.h"
 
+#include "services\TextReader.hh"
+
 class RA_RichPresenceInterpreter
 {
 public:
     RA_RichPresenceInterpreter() {}
```

Comment:
Not a big deal, but since you're already here this should be marked as `default `and `noexcept`. Another option could be just to get rid of it for now.

Reply:
done